src/devhub/entity/community/Discussions.jsx

```diff
@@ -0,0 +1,198 @@
+
+const communityData = getCommunity({ handle });
+
+const MainContent = styled.div`
+  padding-left: 2rem;
+  flex: 3;
+  @media screen and (max-width: 960px) {
+    padding-left: 0rem;
+  }
+  .post:hover {
+    background-color: inherit !important;
+  }
+`;
+
+const SidebarContainer = styled.div`
+  flex: 1;
+`;
+
+const Heading = styled.div`
+  font-size: 19px;
+  font-weight: 600;
+`;
+
+const SubHeading = styled.div`
+  font-size: 15px;
+  font-weight: 600;
+`;
+
+const Container = styled.div`
+  flex-wrap: no-wrap;
+  max-width: 100%;
+
+  .max-width-100 {
+    max-width: 100%;
+  }
+  @media screen and (max-width: 960px) {
+    flex-wrap: wrap;
+  }
+
+  .card {
+    border-radius: 1rem !important;
+  }
+`;
+
+const Tag = styled.div`
+  border-top-right-radius: 50px;
+  border-bottom-right-radius: 50px;
+  border-top-left-radius: 50px;
+  border-bottom-left-radius: 50px;
+  padding-inline: 0.8rem;
+  padding-block: 0.3rem;
+  display: flex;
+  gap: 0.5rem;
+  border-width: 1px;
+  border-style: solid;
+  font-size: 14px;
+  color: rgba(0, 236, 151, 1);
+  font-weight: 800;
+`;
+
+const [sort, setSort] = useState("timedesc");
+
+let grantNotify = Near.view(
+  "${REPL_SOCIAL_CONTRACT}",
+  "is_write_permission_granted",
+  {
+    predecessor_id: "${REPL_DEVHUB_LEGACY}",
+    key: context.accountId + "/index/notify",
+  }
+);
+
+let grantRepost = Near.view(
+  "${REPL_SOCIAL_CONTRACT}",
+  "is_write_permission_granted",
+  {
+    predecessor_id: "${REPL_DEVHUB_LEGACY}",
+    key: context.accountId + "/index/repost",
+  }
+);
+
+if (grantNotify === null || grantRepost === null) {
+  return;
+}
+
+return (
+  <div className="w-100" style={{ maxWidth: "100%" }}>
+    <Container className="d-flex gap-3 m-3 pl-2">
+      <MainContent className="max-width-100">
+        <div className="d-flex flex-column gap-4">
+          {context.accountId && (
+            <div className="card p-4">
+              <Widget
+                src={"${REPL_DEVHUB}/widget/devhub.entity.community.Compose"}
+                props={{
+                  onSubmit: (v) => {
+                    let createDiscussionTx = [
+                      {
+                        contractName: "${REPL_DEVHUB_CONTRACT}",
+                        methodName: "create_discussion",
+                        args: {
+                          handle,
+                          data: v,
+                        },
+                        gas: Big(10).pow(14),
+                      },
+                    ];
+
+                    if (grantNotify === false || grantRepost === false) {
+                      createDiscussionTx.unshift({
+                        contractName: "${REPL_SOCIAL_CONTRACT}",
+                        methodName: "grant_write_permission",
+                        args: {
+                          // @ailisp this shouldn't be legacy right? because with notification it is
+                          predecessor_id: "${REPL_DEVHUB_CONTRACT}",
```

Comment:
@ailisp This should be `devhub.near` right? because with notification in `entity/post/Post.jsx` it is `devgovgigs.near`.